editor/editor_inspector.cpp

```diff
@@ -2851,7 +2821,7 @@ void EditorInspector::update_tree() {
 			if (!EditorNode::get_editor_data().is_type_recognized(type) && p.hint_string.length() && ResourceLoader::exists(p.hint_string)) {
 				Ref<Script> scr = ResourceLoader::load(p.hint_string, "Script");
 				if (scr.is_valid()) {
-					StringName script_name = EditorNode::get_editor_data().script_class_get_name(scr->get_path());
+					String script_name = EditorNode::get_editor_data().script_class_get_name(scr->get_path());
```

Comment:
This is a more of personal idea I had: make the categories showing script file names be shown quoted ("for_example.gd"). Thinking again, this likely a bad idea, so I will be reverting it.

Reply:
There may be merit in it, though I'd rather we made it a clear change with a discussion and a dedicated PR. It's bikeshed material, so I'm tempted to say "let's just merge and see if anyone complains", but at the same time I'm not entirely convinced it would make things better?.. So yeah.